### PR TITLE
Remove electron-log from preload (cherry pick for #1673)

### PR DESCRIPTION
**Pull request by Willyfrog**
#### Summary

manu cherry pick for #1673 

#### Device Information
This PR was tested on: macOS BigSur
#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: src/main/preload/mattermost.js [hunk 1]
```diff
@@ -7,21 +7,18 @@
...
 
 import {ipcRenderer, webFrame} from 'electron';
-import log from 'electron-log';
 
 import {NOTIFY_MENTION, IS_UNREAD, UNREAD_RESULT, SESSION_EXPIRED, SET_SERVER_NAME, REACT_APP_INITIALIZED, USER_ACTIVITY_UPDATE} from 'common/communication';
 
 const UNREAD_COUNT_INTERVAL = 1000;
 const CLEAR_CACHE_INTERVAL = 6 * 60 * 60 * 1000; // 6 hours
 
-Reflect.deleteProperty(global.Buffer); // http://electron.atom.io/docs/tutorial/security/#buffer-global
```
Comment (hmhealey) [inline src/main/preload/mattermost.js:17]: Why is this being removed?

Comment (devinbinnie) [reply src/main/preload/mattermost.js:17]: Cherry-pick issue

Comment (Willyfrog) [reply src/main/preload/mattermost.js:17]: it is failing stating that global.buffer doesn't exist anymore, also tryng that url doesn't exist anymore 🤷

Comment (devinbinnie) [reply src/main/preload/mattermost.js:17]: I think it was used by the `remote` module, which we don't use anymore.

**Review by devinbinnie — APPROVED** [review]

**Pull request merged**